Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -424,10 +424,10 @@ namespace O3DE::ProjectManager
                     return;
                 }
 
-                auto cmdPath = AZ::IO::FixedMaxPathString::format(
-                    "%s --regset=\"/Amazon/AzCore/Bootstrap/project_path=%s\"", editorExecutablePath.c_str(),
-                    fixedProjectPath.c_str());
-
+                AZStd::vector<AZStd::string> cmdPath = AZStd::vector<AZStd::string>{ 
```

Comment:
optimization: This can be `AZStd::move`d into the m_commandlineParameters variable on line 432.